public/js/silder.js

```diff
@@ -0,0 +1,39 @@
+var slideIndex = 1;
+showSlides(slideIndex);
+var prev = document.querySelector('.prev');
+var next = document.querySelector('.next');
+var dot = document.querySelector('.dot');
+// dot 
+
+prev.addEventListener('click', ()=>{
+    showSlides((slideIndex += -1));  
+})
+
+next.addEventListener('click', ()=>{
+    showSlides((slideIndex += -1));  
+})
+
+
+function currentSlide(n) {
+  showSlides((slideIndex = n));
+}
+
+  function showSlides(n){
+  var i;
+  var slides = document.getElementsByClassName("mySlides");

```

Comment:
Learn to work within a DOM context else you will not be able to fix the problem highlighted into above comment.

Reply:
Done